source/cryxtels.cpp

```diff
@@ -2680,7 +2680,7 @@ void Pixel (int typ)
                                     _z = p2 - p3*180 + p6*p3;
                                     _ox = p0; _oz = _z - p6*p3;
                                     for (c=p6; c<=360; c+=p6) {
-                                        _x = tsin[c] * p4;
+                                        _x = p0 + tsin[c] * p4;
```

Comment:
I might need some help figuring out the differences in rendering. :sweat_smile: The only pixel type included by default with a wave is type 19, and it does not seem to reproduce the issue. Maybe they need to have other parameters for the effect to be noticeable?
With the upstream version:
![before](https://github.com/user-attachments/assets/ce6b3604-310d-4446-bede-52d7b259b7cb)

With these changes:
![after](https://github.com/user-attachments/assets/9870135d-6fde-43cc-a803-52817f80b9d7)